src/utils/is-tty.js

```diff
@@ -0,0 +1,10 @@
+"use strict";
+
+const thirdParty = require("../common/third-party");
+
+// Some CI pipelines incorrectly process.stdout.report isTTY status,
```

Comment:
Looks like some words have been scrambled out of order?

Reply:
😅 true